lib/presenters/show_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class ShowPresenter
+  def initialize(dashboard, resource)
+    @dashboard = dashboard
+    @resource = resource
+  end
+
+  def page_title
+    resource.public_send(dashboard.title_attribute)
+  end
+
+  def attribute_names
```

Comment:
Since this isn't all of the attribute names, it might make sense to call it `secondary_attribute_names` or something similar.